### ISSUE TITLE
# rubocop:disable Style/Documentation will be matched as valid documentation

### ISSUE THREAD

**Issue by PascalTurbo** [bbatsov/rubocop#3807]
## Expected behavior

When using a rubocop:disable tag, it shouldn't be matched as a valid documentation

## Actual behavior

When using a rubocop:disabled tag it is detected as documentation which leads to a chicken-and-egg question.

## Steps to reproduce the problem

```ruby
  # rubocop:disable Style/Documentation
  # rubocop:disable Style/DocumentationMethod
  class Time
    def self.parse(t)
      # do something
    end
  end
```

This example will lead to a `Unnecessary disabling of Style/Documentation.` Message because rubocop thinks, that the disable part is a valid Documentation of that part of code.

## RuboCop version

```
$ rubocop -V
0.46.0 (using Parser 2.3.3.1, running on ruby 2.3.0 x86_64-darwin15)
```
